cmd/integrationArtifactUnDeploy_test.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type integrationArtifactUnDeployMockUtils struct {
+	*mock.ExecMockRunner
+	*mock.FilesMock
+}
+
+func newIntegrationArtifactUnDeployTestsUtils() integrationArtifactUnDeployMockUtils {
+	utils := integrationArtifactUnDeployMockUtils{
+		ExecMockRunner: &mock.ExecMockRunner{},
+		FilesMock:      &mock.FilesMock{},
+	}
+	return utils
+}
```

Comment:
Same question here- do we need utils? Is this for the future?

Reply:
this is piper generated code, not added by me